atcart_omni:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  front_left_drive_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: front_left_wheel_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  front_right_drive_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: front_right_wheel_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  front_left_steering_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: front_left_holder_joint
    pid: {p: 100.0, i: 0.0, d: 10.0}

  front_right_steering_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: front_right_holder_joint
    pid: {p: 100.0, i: 0.0, d: 10.0}


  rear_left_drive_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: rear_left_wheel_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  rear_right_drive_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: rear_right_wheel_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  rear_left_steering_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: rear_left_holder_joint
    pid: {p: 100.0, i: 0.0, d: 10.0}

  rear_right_steering_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: rear_right_holder_joint
    pid: {p: 100.0, i: 0.0, d: 10.0}


